database/firebird/firebird_test.go

```diff
@@ -36,8 +38,8 @@ var (
 		},
 	}
 	specs = []dktesting.ContainerSpec{
-		{ImageName: "jacobalberty/firebird:2.5-ss", Options: opts},
-		{ImageName: "jacobalberty/firebird:3.0", Options: opts},
+		{ImageName: "jacobalberty/firebird:2.5.8-ss", Options: opts},
+		{ImageName: "jacobalberty/firebird:3.0.4", Options: opts},
```

Comment:
Let's leave the the version as 3.0.
https://hub.docker.com/r/jacobalberty/firebird/tags

It looks like only a single tag is updated per release: https://github.com/jacobalberty/firebird-docker/blob/master/hooks/post_push